RecoBTag/ONNXRuntime/python/pfParticleNet_cff.py

```diff
@@ -45,6 +67,9 @@
 # nominal: probs
 _pfParticleNetJetTagsProbs = ['pfParticleNetJetTags:' + flav_name
                               for flav_name in pfParticleNetJetTags.flav_names]
+
+_pfParticleNetSonicJetTagsProbs  = ['pfParticleNetSonicJetTags:' + flav_name
+                                    for flav_name in pfParticleNetSonicJetTags.flav_names]
```

Comment:
just reuse `pfParticleNetJetTags` here

Reply:
Do you mean have:
_pfParticleNetSonicJetTagsProbs  = ['pfParticleNetSonicJetTags:' + flav_name
                                    for flav_name in pfParticleNetJetTags.flav_names]

Reply:
yes, or even `_pfParticleNetSonicJetTagsProbs = [x.replace('pfParticleNet','pfParticleNetSonic') for x in _pfParticleNetJetTagsProbs`]

Reply:
I went with _pfParticleNetSonicJetTagsProbs = ['pfParticleNetSonicJetTags:' + flav_name
for flav_name in pfParticleNetJetTags.flav_names], if that's ok.  Just seems a little easier to read to me, but I'm totally willing to change it to your suggestion if preferred